src/resources/wifi_espnow_esp32.cc

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2024 Toitware ApS.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; version
+// 2.1 only.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// The license can be found in the file `LICENSE` in the top level
+// directory of this repository.
+
+#include "../top.h"
+
+#ifdef TOIT_ESP32
+
+#include "wifi_espnow_esp32.h"
+#include "../resource_pool.h"
+
+namespace toit {
+#if defined(CONFIG_TOIT_ENABLE_WIFI) || defined(CONFIG_TOIT_ENABLE_ESPNOW)
```

Comment:
I think I'd prefer to compile out the whole file if the configs are not enabled, so maybe move this earlier?

Reply:
done.